aict_tools/plotting.py

```diff
@@ -3,17 +3,19 @@
 import matplotlib
 import matplotlib.pyplot as plt
 from matplotlib.colors import LogNorm
+from aict_tools.cta_helpers import horizontal_to_camera_cta_simtel
```

Comment:
This now only works for `disp_config.coordinate_transform = 'CTA'`, have a look into the train disp script on how to do it so that it works for both FACT and CTA.

Reply:
Done